mainnet/xion-mainnet-1/README.md

```diff
@@ -1,4 +1,4 @@
-![Burnt banner](https://files.xion-testnet-1.burnt.com/banner.jpg)
+![Burnt banner](https://files.xion-mainnet-1.burnt.com/banner.jpg)
```

Comment:
this doesn't exist. revert to https://files.xion-testnet-1.burnt.com/banner.jpg

Reply:
fixed!!